### DISCUSSION TITLE
Is this scam?

### DISCUSSION THREAD

**Discussion by gurzelai** [community/community#109357] [Other Feature Feedback, Questions, & Ideas] [answered]
I received an email from notifcations@github.com with the same comment that was done in this issue 20 minutes ago.
![image](https://github.com/community/community/assets/80177434/2dd46d7f-a79d-4353-a1d0-1a11a65a1c89)

**Comment by cgtechcommunity** [comment]
Please see: https://github.com/orgs/community/discussions/109171

**Reply by cococo2000** [reply]
404, Page not Found. It's quite peculiar that this discussion is still ongoing in the afternoon......

**Reply by DeandrePetersen** [reply]
It's a topic where another member of github met with the same problem. It's really a scam, so be careful with this emails, alwasy check the address link and also contact customer help if you have any doubts.

**Reply by elGuille** [reply]
What the actual super fuck? 
The original person warning us about this scam got their account suspended?? But the scammer didn't??
![ice_cube_wtf](https://github.com/community/community/assets/1398819/db6faea4-21b9-4017-8a69-2bc27fe603a4)

**Reply by Lanrad23** [reply]
I also received something like this. It was a while ago when I was job hunting and sent my resume to a countless number of companies.
Well, now it's clear why this ""company"" never contacted me back. That's horrible, and I hope you can fix it.

**Comment by ettaboyle** [comment, marked as answer]
Thank you for sharing this @gurzelai

_I can't share specific details on why the message linked above is no longer available, but here is the updated guidance we're sharing broadly. You may see this commented on similar discussions by other members of our team._

We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by CodeLife234** [comment]
**NOTE: Raising assistance on behalf of my friend's flagged GitHub account after applying to that spam mail**

I want to bring to your attention a concerning incident involving my GitHub account (https://github.com/WayneMusungu) that occurred on Wednesday. After interacting with that link, my account was compromised and subsequently flagged. Unauthorized access led to the creation of two repositories named "backup29d8b7f9-0562-45c3-a3d8-29e835a0a467" and "backupc061fd12-0a4d-480b-b3a0-41994af140b2." Adding to the severity of the situation, a message was left stating, "I hope this message finds you well. This is an urgent notice to inform you that your data has been compromised, and we have secured a backup with this link [https://telegra.ph/URGENT-NOTICE-02-01."](https://telegra.ph/URGENT-NOTICE-02-01.%22)
 
Furthermore, my account and projects are no longer publicly visible, and certain repositories have been deleted. This incident transpired after interacting with that spam recruiting email, leading to the compromise of my account.
 
I am seeking prompt assistance from the community in resolving this matter. Your support and guidance would be highly appreciated. I have raised the issue with GitHub, and the ticket number is [Ticket 2619912](https://support.github.com/ticket/2619912). Please be cautious and do not fall for this scam, as I am still unable to access my account. I have also attempted to contact GitHub support through a friend's account.
 
@ettaboyle Your assistance in recovering my account would be greatly valued.
 
Best regards,
Wayne,

**Reply by Lanrad23** [reply]
This actually looks terrifying. Now, I'm a bit worried because a while ago, I actually responded to that scam email. Who knows what those scammers could do to my account?

**Comment by CodeLife234** [comment]
> **NOTE: Raising assistance on behalf of my friend's flagged GitHub account after applying to that spam mail**
> 
> I want to bring to your attention a concerning incident involving my GitHub account (https://github.com/WayneMusungu) that occurred on Wednesday. After interacting with that link, my account was compromised and subsequently flagged. Unauthorized access led to the creation of two repositories named "backup29d8b7f9-0562-45c3-a3d8-29e835a0a467" and "backupc061fd12-0a4d-480b-b3a0-41994af140b2." Adding to the severity of the situation, a message was left stating, "I hope this message finds you well. This is an urgent notice to inform you that your data has been compromised, and we have secured a backup with this link [https://telegra.ph/URGENT-NOTICE-02-01."](https://telegra.ph/URGENT-NOTICE-02-01.%22)
> 
> Furthermore, my account and projects are no longer publicly visible, and certain repositories have been deleted. This incident transpired after interacting with that spam recruiting email, leading to the compromise of my account.
> 
> I am seeking prompt assistance from the community in resolving this matter. Your support and guidance would be highly appreciated. I have raised the issue with GitHub, and the ticket number is 2619912. Please be cautious and do not fall for this scam, as I am still unable to access my account. I have also attempted to contact GitHub support through a friend's account.
> 
> @ettaboyle Your assistance in recovering my account would be greatly valued.
> 
> Best regards, Wayne,

I had initially commented that I can't access the forms for application
![MicrosoftTeams-image](https://github.com/community/community/assets/153800509/c69f2dfe-fd17-4b95-a424-768ec21c8a21)


Then later I saw my account hacked and two repositories created.

![MicrosoftTeams-image (1)](https://github.com/community/community/assets/153800509/f277710b-4df0-44b4-a3aa-2c6741230fc3)

**Comment by K-Naveenkumar1** [comment]
response1
